src/ui/Tab/ResponsiveTabs.ts

```diff
@@ -0,0 +1,178 @@
+import {$$, Dom} from '../../utils/Dom.ts';
+import {InitializationEvents} from '../../events/InitializationEvents.ts';
+import {PopupUtils, HorizontalAlignment, VerticalAlignment } from '../../utils/PopupUtils';
+
+export class ResponsiveTabs {
+  static coveoTabSection = '.coveo-tab-section';
+  
+  private static dropdownHeader: Dom;
+  private static dropdownContent: Dom;
+  private static tabSection: Dom;
+  private static searchBarElement: HTMLElement;
+  private static coveoRoot: HTMLElement;
+  
+  public static init() {
+    this.coveoRoot = <HTMLElement>document.querySelector('.CoveoSearchInterface');
```

Comment:
Cannot do this : It's very possible that there is multiple search interface in the same page.